tally_ho/apps/tally/models/result_form.py

```diff
@@ -422,6 +423,61 @@ def reject(
         self.reject_reason = reject_reason
         self.save()
 
+    def reset_to_unsubmitted(self, user=None, reason=None):
```

Comment:
Can we consider wrapping the function in a `@transaction.atomic` decorator to ensure all the changes are applied or none is applied in the event of a failure in any one of them?

Reply:
done